doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -381,7 +381,11 @@ Modem libraries
 
 * :ref:`nrf_modem_lib_readme`:
 
-  * Updated the :c:func:`nrf_modem_lib_shutdown` function to allow the modem to be in flight mode (``CFUN=4``) when shutting down the modem.
+  Updated:
+
+    * The :c:func:`nrf_modem_lib_shutdown` function to allow the modem to be in flight mode (``CFUN=4``) when shutting down the modem.
+    * The trace backends can now return ``-EAGAIN`` if the write operation can be retried.
+    * The trace backends ``deinit`` function is expected to succeed and can no longer return an error.
```

Comment:
@eivindj-nordic : There is one more entry : _Added CEREG event tracking to lte_connectivity_ that should be part of :ref:`nrf_modem_lib_readme`: . Could you move this entry here.

Thanks for fixing this.

Reply:
done